tests/link/jax/test_slinalg.py

```diff
@@ -122,6 +122,38 @@ def test_jax_solve():
     )
 
 
+def test_jax_tridiagonal_solve():
```

Comment:
Parameterize this to test the following shapes:

1. A = (N, N), b = (N,), b_ndim = 1 (basic vector case)
2. A = (N, N), b = (N, M), b_ndim = 2 (basic matrix case)
3. A = (N, N), b = (M, N), b_ndim = 1 (vector case with broadcasting)
4. A = (D, N, N), b = (D, M, N), b_ndim = 2 (fully batched case)

Reply:
done